Functions/Quantification/xASL_quant_ASL.m

```diff
@@ -114,7 +116,7 @@
             if bUseBasilQuantification
                 x.Q.BasilSliceReadoutTime = 0;
             else
-                ScaleImage = ScaleImage.*x.Q.Initial_PLD;
+                ScaleImage = ScaleImage.*x.Q.uniqueInitial_PLD;
```

Comment:
This will be a vector for multi-PLD, right?

Reply:
Yes, it needs to be initialized at the start of xASL_module_ASL

Reply:
Good point, some parameters may need to become 4D instead of 3D. Let's discuss how we can do this in the best way.

Reply:
That's not true, because we have the ability to provide CBF4D, and in the future we could have non-singlePLD quantified by us. So it is probably best to have ScaleImage always in 4D, we would have to discuss this.